interface/patient_file/encounter/Services/ViewHelper.php

```diff
@@ -0,0 +1,264 @@
+<?php
+/**
+ * Helper functions for an encounter view files
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option)
+ * any later version. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details. You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @subpackage Encounter
+ * @license GNU GPL3
```

Comment:
this could get sticky though. For example, what if we want to update to GNU GPL4 if there is ever a GPL4 of course (the header does it automatically, but can we change the tag or does the main author need to do it; etc. etc.). Just seems best to avoid these issues. Note I am biased since we have had to deal with several licensing confusions in the older codebase:
http://www.open-emr.org/wiki/index.php/Active_Projects#Attribution.2C_licensing_and_copyright_clarification